lib/Perl/MinimumVersion.pm

```diff
@@ -870,6 +871,49 @@ sub _get_resulting_sigil {
 	}
 }
 
+# Postfix dereference new (and experimental) in 5.20, mainstream in 5.24.
+# THIS CODE ASSUMES PPI 1.237_001 OR ABOVE -- i.e. support for postfix
+# dereferencing.
+#
+my %postfix_deref = (
+	'$*'	=> \&_postfix_deref_entire,
+	'@*'	=> \&_postfix_deref_entire,
+	'$#*'	=> \&_postfix_deref_entire,
+	'%*'	=> \&_postfix_deref_entire,
+	'&*'	=> \&_postfix_deref_entire,
+	'**'	=> \&_postfix_deref_entire,
```

Comment:
Confirming that this matches all the possible "circumfix-equivalent" postfix derefs <https://perldoc.perl.org/5.36.1/perlref#Postfix-Dereference-Syntax>.